style.css

```diff
@@ -188,3 +188,39 @@ span.has-inline-color > strong {
 	overflow: hidden;
 	padding: 1.25em 2.375em;
 }
+
+.wp-block-latest-posts__list > li > a {
+	font-weight: bold;
+	text-decoration: none;
+	color: #000000;
+	font-size: 2.4em;
+	display: inline-block;
+	line-height: 1em;
+}
+
+.wp-block-latest-posts__post-author {
+	display: inline-block;
+	font-weight: bold;
+	color: #c74200;
+	font-size: 1rem;
+}
+
+.wp-block-latest-posts__post-date {
+	display: inline-block;
+	font-weight: bold;
+	font-size: 1rem;
+	margin-left: 0.4rem;
+}
+.wp-block-latest-posts__post-date::before {
+	content: '\2022';
+	font-size: 2rem;
+	color: #B0B0B0;
+	margin-right: 0.4rem;
+	margin-top: 0.5rem;
+	top: 0.1em;
+    position: relative;
+}
+
+.wp-block-latest-posts__featured-image {
+	float: left;
+}
```

Comment:
I believe we should avoid hard-coding this float rule since it conflicts with the Gutenberg Latest Posts block Featured Image alignment settings:

![image](https://user-images.githubusercontent.com/17307/142851105-5ddb2ac4-b0b4-488b-ad8c-f48f8018099b.png)
```suggestion
```